apps/chat-e2e/src/tests/chatExportImport.test.ts

```diff
@@ -129,6 +129,9 @@ dialTest(
         await dialHomePage.importFile(exportedData, () =>
           chatBar.importButton.click(),
         );
+        await folderConversations.expandFolder(
+          conversationInFolder.folders.name,
+        );
```

Comment:
according to requirements, the folder should be autoexpanded after import. Please, fix this behavior and remove additional steps from tests